tests/unittests/core/conftest.py

```diff
@@ -114,3 +114,23 @@ def hacked_exp(with_user_dendi, random_dt, clean_db):
     exp = Experiment('supernaedo2')
     exp._id = 'supernaedo2'  # white box hack
     return exp
+
+
+@pytest.fixture()
+def trial_id_substitution(with_user_tsirif, random_dt, clean_db):
```

Comment:
omg yes.

Reply:
hacked_exp is deprecated now

Reply:
or it could be considered the same but from the opposite approach.

Reply:
I don't know if it's worth removing unit-tests with hacked_exp, but yes this fixture kind of make hacked_exp deprecate. There is a similar fixture but for experiment cross references in #94 .